packages/core/test/UserState/userStateTransitionEvent.test.ts

```diff
@@ -36,6 +32,7 @@ describe('User state transition events in Unirep User State', async function ()
     let attester: any
     const maxUsers = 2 ** 7
     const attestingFee = ethers.utils.parseEther('0.1')
+    let numEpochKeyNoncePerEpoch
```

Comment:
This doesn't need to be stored globally, it can be accessed on a Synchronizer or UserState object as `userState.settings.numEpochKeyNoncePerEpoch`. It can be loaded manually using a single line `await unirepContract.numEpochKeyNoncePerEpoch()`